packages/scandipwa/src/component/Header/Header.component.js

```diff
@@ -325,9 +326,16 @@ export class Header extends NavigationAbstract {
     renderShareWishListButton(isVisible = false) {
         const {
             isWishlistLoading,
-            shareWishlist
+            shareWishlist,
+            productsInWishlist
         } = this.props;
 
+        const wishListQty = Object.keys(productsInWishlist).length;
+
+        if (!wishListQty) {
```

Comment:
perhaps there is no point to create variable for 1 time use in if block condition.